beeai/agents/rebase_agent.py

```diff
@@ -66,6 +70,7 @@ def __init__(self) -> None:
             memory=UnconstrainedMemory(),
             requirements=[
                 ConditionalRequirement(ThinkTool, force_after=Tool, consecutive_allowed=False),
+                ConditionalRequirement("build_package", min_invocations=1),
```

Comment:
I like the idea of enforcing this, but on the other hand, if the update was already done, which we check in the beginning and skip the update for that case, it wouldn't be correct; I am wondering if this could be still somehow conditionally enforceed